docs/multi-manager-setup.md

```diff
@@ -11,15 +11,15 @@ weight=3
 
 # High availability in Docker Swarm
```

Comment:
OK. As we are late in the game. I'm gonna remove the HA work.  We can test and add it back in later.